LeaderboardBackend/Program.cs

```diff
@@ -136,7 +136,9 @@
         // Enforces JSON output and causes OpenAPI UI to correctly show that we return JSON.
         opt.OutputFormatters.RemoveType<StringOutputFormatter>();
         opt.ModelBinderProviders.Insert(0, new UrlSafeBase64GuidBinderProvider());
+        opt.Filters.AddService<ValidationFilter>();
```

Comment:
This is for my own learning; do we need to call `AddSingleton` (or basically add the service to `builder.Services` as you've done below whenever we add a new entry here?

Reply:
Only if it's a service